packages/harpguru-core/src/components/menu/menu.tsx

```diff
@@ -25,8 +25,8 @@ export const Menu = ({
   const scaledLabelProtrusion = useScaledMenuLabelProtrusion()
 
   const {
-    dynamicSizes: { 9: borderRadius },
-  } = useSizes()
+    [0]: { 9: borderRadius },
+  } = useGlobal('dynamicSizes')
```

Comment:
Similar point to https://github.com/js-jslog/harpguru/pull/137#discussion_r765155671

Reply:
Done